redaxo/src/addons/structure/plugins/version/boot.php

```diff
@@ -22,7 +22,7 @@
     rex_login::startSession();
 
     if (!rex_backend_login::hasSession()) {
-        throw new rex_exception('no permission for the working version');
+        throw new rex_exception('No permission for the working version. You need to be logged into the REDAXO backend at the same time.');
```

Comment:
Hmm ggf. Wäre es sinnvoll an dieser stelle nen login dialog zu bringen fürs backend?

Ein echter besucher der frontend website sollte ja diese stelle mit diesem rex-version parameter nie erreichen ,oder?

Aktuell gibt es manchmal verwirrungen wenn browser die http session nicht über die tabs hinaus verwenden, oder http vs https oder unterschiedliche domains verwendet werden für FE/BE

Reply:
//cc @gharlan @tbaddade